### PR TITLE
change language codes

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@saeedmb8 Please review

File: iogt/settings/base.py [hunk 1]
```diff
@@ -229,36 +207,6 @@
...
-        'code': 've',
-        'name': 'Venda',
-        'name_local': 'Venda'
-    },
-    'tn': {
-        'bidi': False,
-        'code': 'tn',
-        'name': 'Tswana',
-        'name_local': 'Tswana'
-    },
```
Comment (smarzban) [inline iogt/settings/base.py]: @Ntu2koTrevor lets keep all the languages here, just please make sure all the languages are correct

File: iogt/settings/base.py [hunk 2]
```diff
@@ -271,96 +219,6 @@
...
-        'code': 'dv',
-        'name': 'Dhivehi',
-        'name_local': 'Dhivehi'
-    },
-    'ber': {
-        'bidi': False,
-        'code': 'ber',
-        'name': 'Berber',
-        'name_local': 'Tamaziɣt'
-    },
```
Comment (smarzban) [inline iogt/settings/base.py]: these as well

**Comment by smarzban** [comment]
👍

**Pull request merged**